src/styles/_ulits.scss

```diff
@@ -0,0 +1,2 @@
+$color--hover--link: #00acdc;
+$color--hover--text: #34568b;
```

Comment:
Add newline in the end
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file